scripts/publish-apt-packages.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# This script is used to publish new packages to the CLI APT repository
+# Usage: ./publish-apt-packages.sh
+set -eo pipefail
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+OBJECT_STORAGE_ENDPOINT="https://object.storage.eu01.onstackit.cloud"
+APT_BUCKET_NAME="stackit-cli-apt"
+PUBLIC_KEY_BUCKET_NAME="stackit-public-key"
+PUBLIC_KEY_FILE="key.gpg"
+CUSTOM_KEYRING="custom-keyring"
+APTLY_CONFIG_FILE_PATH="./.aptly.conf"
+
+# Create a local mirror of the current state of the remote APT repository
+printf ">>> Creating mirror \n"
+curl ${OBJECT_STORAGE_ENDPOINT}/${PUBLIC_KEY_BUCKET_NAME}/${PUBLIC_KEY_FILE} >public.asc
```

Comment:
```suggestion
curl ${OBJECT_STORAGE_ENDPOINT}/${PUBLIC_KEY_BUCKET_NAME}/${PUBLIC_KEY_FILE} > public.asc
```
maybe it also works without the space

Reply:
It's my auto-formatter that takes out that space, but yes it works